akvo/rsr/views/my_rsr.py

```diff
@@ -67,6 +69,8 @@ def my_details(request):
 
     change_password_form = PasswordForm(request.user)
 
+    api_key = ApiKey.objects.get_or_create(user=request.user)[0].key
```

Comment:
@damienallen Could you actually change this around a little. E.g.:

```
try:
    api_key = ApiKey.objects.get(user=request.user).key
except ApiKey.DoesNotExist:
    api_key = None
```

And when api_key is None in the template, don't display the ```div```.